packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/BlocksInput/utils/constants.ts

```diff
@@ -1,31 +1,44 @@
-export const codeLanguages: { value: string; label: string }[] = [
+export const codeLanguages: {
+  value: string;
+  label: string;
+  decorate?: string;
+  importPath?: string;
+}[] = [
   {
     value: 'asm',
     label: 'Assembly',
+    decorate: 'asmatmel',
+    importPath: 'prism-asmatmel',
```

Comment:
Can you point me to the place in the code that uses this importPath property? I can't seem to find it

Reply:
yeah, you are right, it is a property that I tried to use to lazy load the imports with no success, I am gonna remove them